FhirToDataLake/docs/Deploy-FhirToDatalake.md

```diff
@@ -54,13 +54,15 @@ This solution enables you to query against the entire FHIR data with tools such
     | Data Start | Start time stamp of the data to be exported. |
     | Data End | Start time stamp of the data to be exported. Leave it empty if you want to periodically export data in real time.  |
     | Container Name | A name for the Storage Account container to which Parquet files will be written. The Storage Account with autogenerated name will automatically be created during the installation. |
-    | Job Concurrency | Concurrent jobs exeucting in parallel. |
-    | Customized Schema Image Reference (new)  | The customized schema image reference for the image on Container Registry. Refer [TemplateManagement](https://github.com/microsoft/FHIR-Converter/blob/main/docs/TemplateManagementCLI.md) for how to manage your template images. |
-    | Filter Config Image Reference (new)  | The filter config image reference for the image on Container Registry. Refer [TemplateManagement](https://github.com/microsoft/FHIR-Converter/blob/main/docs/TemplateManagementCLI.md) for how to manage your template images. |
-    | Filter Scope (new) | For data filtering use. The export scope can be `System` or `Group`. The default value is `System` if no filter is applied.|
-    | Group Id (new) | For data filtering use. If the Filter scope is set as `Group`, you need to fill the group Id, otherwise leave it blank.|
-    | Required Types (new) | For data filtering use. Specify which types of resources will be included. For example, _type=Patient_ would return only patient resources. All resource types will be exported if not specified. Leave it blank if no filter is applied.|
-    | Type Filters (new) | For data filtering use. Use along with the requiredTypes configuration. The value is a comma-separated list of FHIR queries that further restrict the results. All data of requiredTypes will be exported if not specified. Leave it blank if no filter is applied. |
+    | Job Concurrency | Concurrent jobs executing in parallel. |
+    | Scheduler Interval (new) | The scheduler interval. To customize the scheduler, set the value to "customized" and specify the value in Scheduler Crontab Expression below. |
+    | Scheduler Crontab Expression (new) | Use a crontab expression to set the scheduler. A crontab expression is a six-part crontab format (sec min hour day month day-of-week). Refer to [this link](https://github.com/atifaziz/NCrontab) for more about crontab expression. |
```

Comment:
Seems we are using the **NCronTab** expression like the [azure function](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=in-process&pivots=programming-language-csharp#ncrontab-expressions)?